pyomo/solvers/tests/mip/test_ipopt.py

```diff
@@ -204,11 +214,12 @@ def test_bad_dof(self):
         m.c.add(m.x + m.y == 1)
         m.c.add(m.x - m.y == 0)
         m.c.add(2*m.x - 3*m.y == 1)
-        m.write('j.nl')
+        #m.write('j.nl')
```

Comment:
Can this commented line be removed?

Reply:
Yes.  This file was written by the original test and not deleted.  Given that no part of the test checked or did anything with the file, I commented it out, but wanted to leave it in case someone knew why we were generating the file in the first place.  It can be deleted outright if no one objects.